l10n/en/brands.ftl

```diff
@@ -123,7 +123,7 @@
 -brand-name-mozilla-account = Mozilla account
 -brand-name-mozilla-accounts = Mozilla accounts
 -brand-name-mozilla-social = Mozilla.social
--brand-name-mozilla-ai = Mozilla AI
+-brand-name-mozilla-ai = Mozilla.ai
```

Comment:
@craigcook This will need to be versioned(?) https://github.com/mozilla-l10n/www-l10n/pull/456#discussion_r1868528813